ast/dml.go

```diff
@@ -467,24 +468,27 @@ type SelectLockInfo struct {
 }
 
 // String implements fmt.Stringer.
-func (n SelectLockType) String() string {
+func (n SelectLockType) String() (string, error) {
```

Comment:
Returning the error makes it not implement Stringer anymore. Please keep returning string only.